dynamic/longestcommonsubsequence.go

```diff
@@ -36,5 +41,5 @@ func LongestCommonSubsequence(a string, b string, m int, n int) int {
 // 	var a,b string
 // 	fmt.Scan(&a, &b)
 // 	// calling the LCS function
-// 	fmt.Println("The length of longest common subsequence is:", longestCommonSubsequence(a,b, len(a), len(b)))
+// 	fmt.Println("The length of longest common subsequence is:", longestCommonSubsequence(a,b))
```

Comment:
non-blocking, but this whole comment block can be removed 🙂 We have tests now, so they are not needed.